tests/ssid_set_snmp.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2017
+#
+# All rights reserved.
+#
+# This file is distributed under the Clear BSD license.
+# The full text can be found in LICENSE in the root directory.
+
+import os, datetime, pexpect
+import rootfs_boot
+from devices import board, wan, lan, prompt
+from lib.installers import install_snmp
+from cbnlib import now_short
+
+class SSIDSetSnmp(rootfs_boot.RootFSBootTest):
+    '''Setting unique SSID'''
+    log_to_file = ""
+    set_ssid_name = None
+    index = None
+    def runTest(self):
+        install_snmp(wan)
+        wan_ip = board.get_interface_ipaddr(board.wan_iface)
+        """"setting ssid for wifi"""
+        wifi_name = ['wifi_2g','wifi_5g']
+        for wifi_device in wifi_name:
+            if wifi_device == 'wifi_2g':
+                self.index = '.32'
+                self.set_ssid_name = 'SSIDwifi_2G'
```

Comment:
These wifi names will need to contain more than a hard coded name. Otherwise we will see so many of these. I suggest using a station prefix with a random SHA as well.